borg/helpers.py

```diff
@@ -143,13 +144,14 @@ def update(self, size, csize, unique):
 
     def print_(self, label, cache):
         total_size, total_csize, unique_size, unique_csize, total_unique_chunks, total_chunks = cache.chunks.summarize()
```

Comment:
it seems like at that place there should be newstyle string formating and a namedtuple return value